Server/mods/deathmatch/logic/CVehicle.cpp

```diff
@@ -644,7 +647,7 @@ bool CVehicle::SetOccupant(CPed* pPed, unsigned int uiSeat)
         return true;
     }
 
-    return true;
+    return true; // return true? even if function didn't end successfully?
```

Comment:
It ends successfully. That `bAlreadySetting` act as lock to prevent infinite recursion.